crates/autopilot/src/domain/auction/order.rs

```diff
@@ -12,6 +15,7 @@ pub struct Order {
     pub protocol_fees: Vec<fee::Policy>,
     pub side: Side,
     pub class: Class,
+    pub created: u32,
```

Comment:
In the spirit of domain driver design we should use a nicer timestamp type than `u32`. (also applies to `valid_to`)

Reply:
But `valid_to` is stored as BigInt in the DB. Also, Felix previously [suggested](https://github.com/cowprotocol/services/pull/2855#discussion_r1701689395) keeping the types aligned.